api/views.py

```diff
@@ -26,10 +32,29 @@ class Meta:
         model = Classifier
         fields = ['user', 'created_at', 'updated_at']
 
-class ClassifierCreate(generics.CreateAPIView):
+class ClassifierCreate(generics.CreateAPIView, mixins.CreateModelMixin):
     permission_classes = (ClassifierCreatePermission,)
     serializer_class = ClassifierSerializer
 
+    def add_user_to_mailchimp_list(self, email_address, mailchimp_api_key):
+        requests.post(MAILCHIMP_LIST_URL,
+            json={"email_address": email_address,
+                  "status": "subscribed"},
+            # If we can't get this from the environment,
+            # return 500 so we know something is broken.
```

Comment:
We actually are properly checking if we can in fact get this from the environment so no 500s will be returned. Basically, you can take out this comment.